### ISSUE TITLE
hi,I trained your code with my own data, but the loss value is nan, when the program is running.Do you know what is happened to this.

### ISSUE THREAD

**Issue by zhizhongchai** [ankurhanda/tf-unet#1]

**Comment by ankurhanda** [comment]
check if the number of labels is more than the number of output channels from the softmax

**Comment by weisq2691** [comment]
maybe your lr  is too big

**Comment by ankurhanda** [comment]
I've done a major overhauling of the code and this is far better. @zhizhongchai @weisq2691
